packages/helm-charts/oracle/templates/azure-identity-binding.yaml

```diff
@@ -1,4 +1,5 @@
 {{- range $index, $identity := .Values.oracle.identities -}}
+{{ if (hasKey $identity "azure") }}
```

Comment:
It feels like these should be indented, if not just for legibility

Reply:
I definitely agree it's ugly, but as far as I know helm can complain if there are indentations because yaml relies on whitespace